src/main/scala/viper/silver/cfg/silver/SilverCfg.scala

```diff
@@ -87,7 +87,7 @@ class SilverCfg(val blocks: Seq[SilverBlock], val edges: Seq[SilverEdge], val en
     * Recursively finds the next join point to a branch point.
     *
     * @param queueInit Initializes the BFS queue with blocks that are yet to be visited.
-    * @param visitedInit Initialized list of visited nodes.
+    * @param visitedInit Initializes list of visited nodes.
```

Comment:
Are parameters `queueInit` and `visitedInit` effectively out-parameters that are initialised by the function (and hence "initialises" rather than "initialised")?

Reply:
Oh I didn't realize that this small code comment change may cause confusion. Both `queueInit` and `visitedInit` are really just for initializing the the queue and the visited set. They are not used as out-parameters.

Reply:
Thanks, that clarifies it. How about a phrasing along the lines of "Used to initialize the ..." or "Blocks to initialize the ... with"?